linera-execution/src/runtime.rs

```diff
@@ -776,6 +795,29 @@ impl<UserInstance> BaseRuntime for SyncRuntimeInternal<UserInstance> {
         Ok(value)
     }
 
+    fn contains_keys_new(
+        &mut self,
+        keys: Vec<Vec<u8>>,
+    ) -> Result<Self::ContainsKeys, ExecutionError> {
+        let id = self.application_id()?;
+        let state = self.view_user_states.entry(id).or_default();
+        self.resource_controller.track_read_operations(1)?;
```

Comment:
@afck This addition makes me feel that we should count the number of keys (here and below).